### PR TITLE
Feature/jenkins 38597 create from git ath

### PR DESCRIPTION
**Pull request by cliffmeyers**
# Description
- Basic sanity test for Git Creation. As the validation flow is going to change slightly after #759 we are deferring involved ATH until then.
- Relies on change in jenkinsci/blueocean-plugin#780
- See [JENKINS-38597](https://issues.jenkins-ci.org/browse/JENKINS-38597).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

**Comment by michaelneale** [comment]
:bee:

**Comment by michaelneale** [comment]
Looks good - tried it, did see the odd timing failure: 

```
 Timed out while waiting for element <.activity-table tr#test-project-folder-1 svg path.running> to be present for 20000 milliseconds.  - expected "found" but got: "not found"
```

My suggestion is in this case to test for the end case, vs the intermediate - ie if the result is that it has built - just look for that. That way less sensitive to timing, and this test is to cover creation, not loading of activity screen and timing.

File: src/test/js/creation/git/localRepo.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        create.waitForElementVisible('.git-step-connect');
+        create.setValue('@repositoryUrlText', pathToRepo);
+        create.click('@newCredentialTypeSystemSSh');
+        create.click('@createButton');
+        create.assertCompleted();
+    },
+    'Step 02 - Check Activity Tab': function (browser) {
+        const activity = browser.page.bluePipelineActivity().forJob(jobName, 'jenkins');
+        activity.assertBasicLayoutOkay();
+        activity.waitForRunRunningVisible(jobName + '-1');
```
Comment (michaelneale) [inline src/test/js/creation/git/localRepo.js]: As mentioned in comment, why not use: 

```
activity.waitForRunSuccessVisible(jobName + '-1');
```

better results and doesn't care about timing.

**Comment by michaelneale** [comment]
Looks good - tried it, did see the odd timing failure: 

```
 Timed out while waiting for element <.activity-table tr#test-project-folder-1 svg path.running> to be present for 20000 milliseconds.  - expected "found" but got: "not found"
```

My suggestion is in this case to test for the end case, vs the intermediate - ie if the result is that it has built - just look for that. That way less sensitive to timing, and this test is to cover creation, not loading of activity screen and timing.

File: src/test/js/creation/git/localRepo.js (hunk 1 above)
Comment (michaelneale) [inline src/test/js/creation/git/localRepo.js]: As mentioned in comment, why not use: 

```
activity.waitForRunSuccessVisible(jobName + '-1');
```

better results and doesn't care about timing.

Comment (michaelneale) [inline src/test/js/creation/git/localRepo.js]: As mentioned in comment, why not use: 

```
activity.waitForRunSuccessVisible(jobName + '-1');
```

better results and doesn't care about timing.

Comment (cliffmeyers) [inline src/test/js/creation/git/localRepo.js]: Fixed in 82afdb3

**Comment by cliffmeyers** [comment]
@michaelneale this should be ready to go now.

**Comment by michaelneale** [comment]
can go in when correseponding branch in master is.

**Comment by michaelneale** [comment]
:bee: runs nicely against master

**Pull request merged**